CHANGELOG.md

```diff
@@ -7,10 +7,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### ⚠️ Breaking changes
+
+- `ProcessPayload` class renamed to `PayloadManager` to better reflect its purpose. The
+  renamed class is also simplified: `features` and `process` attributes removed,
+  `from_event` method moved to the new `CirrusPayload` class. ([#331])
+
```

Comment:
I'd push back on this being a breaking change. `ProcessPayload` is an internal model that should not be used outside cirrus-geo. If this were not the case and we did have external users I'd agree this change would be breaking. But as it stands I don't think it is breaking, and would rather see this be listed as a non-breaking change. I might also specify this change applies to an internal class, i.e., "Internal `ProcessPayload` class renamed to `PayloadManager`...".

I'd almost go so far as to argue we don't even need to include this change here, but I generally defer to the side of transparency risking useless info than to potentially hide something meaningful from someone.

Reply:
If we cannot agree that this should not be classified as a breaking change then consider that we will need to have a major version bump for this release.